modules/custom/social_demo/src/DemoContentParser.php

```diff
@@ -22,9 +22,19 @@ public function getPath($file, $module, $profile) {
   }
 
   /**
-   * {@inheritdoc}
+   * Parses YAML file into a PHP value.
+   *
+   * @param string $file
+   *   The filename.
+   * @param string $module
+   *   The module where the Yaml file is placed.
+   * @param string $profile
+   *   The profile used.
+   *
+   * @return mixed
+   *   The YAML converted to a PHP value.
    */
-  public function parseFile($file, $module, $profile) {
```

Comment:
@jaapjan I was wondering why this was changed. Is it because of https://www.drupal.org/node/2937545?

Reply:
Correct; that method was added in 3.4 I guess: https://github.com/symfony/yaml/commit/ea18e8d2f6be875713b4bd3db729d600d7f3727b#diff-6beeaadccc3b2cf24ef915866a3b088f 

And since we already used the same method (with different arguments) I thought renaming was the better option.